src/components/sidebar/components/sidebar-nav-menu-item/index.tsx

```diff
@@ -21,13 +21,22 @@ const SidebarNavLink = ({ item }: SidebarNavMenuItemProps) => {
   const href = item.fullPath || item.href
   const isExternal = isAbsoluteUrl(href)
 
+  // TODO: replace any with MenuItem
+  let leadingIcon
+  if ((item as any).leadingIcon) {
+    leadingIcon = (
+      <div className={s.leadingIcon}>{(item as any).leadingIcon}</div>
+    )
+  }
```

Comment:
This is for the top-level mobile Sidebar that shows "Developer Home", "Vault", and "Waypoint" items. If this implementation looks good, I'll make sure to document here and in the props interface what this `leadingIcon` prop is intended for.

![image](https://user-images.githubusercontent.com/43934258/163804395-af885b61-e0b4-446e-a95c-f6e3c8717c8d.png)

Reply:
Added note about this to PR description, resolving! ✨